src/test/java/org/minions/devfund/yuri/linkedlist/RemoveMaximumValuesTest.java

```diff
@@ -0,0 +1,206 @@
+        } catch (Exception var3) {
+            Assert.fail("removeMaximumValues throws " + var3 + " when input LinkedList is empty");
+        }
+
+        Assert.assertTrue("removeMaximumValues adds elements to list when input LinkedList is empty",
+                var1.isEmpty());
+    }
+
+    /**
+     * test.
+     */
+    @Test
+    public void testRemoveGreaterThanSize() {
+        LinkedList var1 = new LinkedList();
+        var1.add("doge");
+
+        try {
+            LinkedListUtils.removeMaximumValues(var1, 2);
+        } catch (Exception var3) {
+            Assert.fail("removeMaximumValues throws " + var3 + " when number of elements to remove is greater than "
+                    + "size of input LinkedList");
+        }
+
+        Assert.assertTrue("removeMaximumValues does not remove all elements when number of elements to "
+                + "remove is greater than size of input LinkedList", var1.isEmpty());
+    }
+
+    /**
+     * test.
+     */
+    @Test
+    public void testRemoveEqualsSize() {
+        LinkedList var1 = new LinkedList();
+        var1.add("doge");
+
+        try {
+            LinkedListUtils.removeMaximumValues(var1, 1);
+        } catch (Exception var3) {
+            Assert.fail("removeMaximumValues throws " + var3 + " when number of elements to remove equals size of "
+                    + "input LinkedList");
+        }
+
+        Assert.assertTrue("removeMaximumValues does not remove all elements when number of elements to "
+                + "remove equals size of input LinkedList", var1.isEmpty());
+    }
+
+    /**
+     * test.
+     */
+    @Test
+    public void testEachMaxAppearsOnce() {
+        LinkedList var1 = new LinkedList();
+        var1.add("doge");
+        var1.add("cat");
+        var1.add("kangaroo");
+        var1.add("koala");
+        var1.add("bear");
+
+        try {
+            LinkedListUtils.removeMaximumValues(var1, 2);
+        } catch (Exception var3) {
+            Assert.fail("removeMaximumValues throws " + var3 + " when number of elements to remove is less than size "
+                    + "of input");
+        }
+
+        if (var1.size() < LISTSIZE1) {
+            Assert.fail("removeMaximumValues removes too many elements from LinkedList when number of elements to "
+                    + "remove is less than size of input");
+        }
+
+        if (var1.size() > LISTSIZE1) {
+            Assert.fail("removeMaximumValues does not remove enough elements from LinkedList when number of elements "
+                    + "to remove is less than size of input");
+        }
+
+        if (var1.contains("kangaroo") || var1.contains("koala")) {
+            Assert.fail("removeMaximumValues does not remove correct elements from LinkedList when number of "
+                    + "elements to remove is less than size of input");
+        }
+
+        Assert.assertTrue("removeMaximumValues does not have elements in correct order when number of "
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "removeMaximumValues does not have elements in correct order when number of " 7 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)

Reply:
Done